s3fs/core.py

```diff
@@ -384,7 +405,7 @@ def close_session(loop, s3):
             try:
                 sync(loop, s3.__aexit__, None, None, None, timeout=0.1)
                 return
-            except TimeoutError:
+            except FSTimeoutError:
```

Comment:
This change, and the import for it, are already in main https://github.com/dask/s3fs/blob/main/s3fs/core.py#L387 , so please merge. This maybe got merged after you posted the PR.

Reply:
Right. Just saw that your PR is merged, so I'll do a rebase. Thanks